components/library/libsecret/Makefile

```diff
@@ -13,39 +13,35 @@
 # Copyright 2016 Alexander Pyhalov
 #
 
```

Comment:
/usr/bin/pkgdepend generate -m -d /export/home/andreas/oi-userland/components/library/libsecret/build/prototype/i386/mangled -d /export/home/andreas/oi-userland/components/library/libsecret/build/prototype/i386 -d /export/home/andreas/oi-userland/components/library/libsecret/build -d /export/home/andreas/oi-userland/components/library/libsecret -d libsecret-0.18.5 /export/home/andreas/oi-userland/components/library/libsecret/build/manifest-i386-libsecret.mangled >/export/home/andreas/oi-userland/components/library/libsecret/build/manifest-i386-libsecret.depend
Couldn't find 'usr/bin/amd64/secret-tool' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libsecret
	/export/home/andreas/oi-userland/components/library/libsecret/build
	/export/home/andreas/oi-userland/components/library/libsecret/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libsecret/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libsecret/libsecret-0.18.5
gmake: *** [/export/home/andreas/oi-userland/make-rules/ips.mk:498: /export/home/andreas/oi-userland/components/library/libsecret/build/manifest-i386-libsecret.depend] Erro